### PR TITLE
feat: misc improvements

### PR DESCRIPTION
**Pull request by 0xArdy**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,10 +29,11 @@
     "@emotion/server": "11.11.0",
     "@emotion/styled": "11.11.0",
     "@eth-optimism/tokenlist": "9.0.25",
+    "@fontsource-variable/roboto-mono": "5.0.18",
```
Comment (turtlemoji) [inline package.json:32]: why a dependency for a font?

File: src/components/MadeByWonderland.tsx [hunk 2]
```diff
@@ -41,6 +41,7 @@ const SBox = styled(Box)(() => {
       width: '13rem',
       filter: logoFilter,
       marginTop: '-0.2rem',
+      marginLeft: '-0.3rem',
```
Comment (turtlemoji) [inline src/components/MadeByWonderland.tsx:44]: everytime a negative margin is used an angel cries

File: src/containers/BridgeCard/TokenSection.tsx [hunk 3]
```diff
@@ -58,6 +58,7 @@ export const TokenSection = () => {
           value={inputValue}
           onChange={handleOnChange}
           fullWidth
+          onKeyDown={(evt) => ['e', 'E', '+', '-'].includes(evt.key) && evt.preventDefault()}
```
Comment (turtlemoji) [inline src/containers/BridgeCard/TokenSection.tsx:61]: abstract to function

File: src/containers/Modals/Review.tsx [hunk 4]
```diff
@@ -29,33 +34,47 @@ export const ReviewModal = () => {
...
+        </span>
       </DataRow>
 
       {/* Fees */}
       <DataRow>
         <Typography variant='body1'>Fees</Typography>
-        <span>$21.33</span>
+        <span>
+          <Image src={gasIcon} alt='fees' />
+          {/* temporary fixed value */}
```
Comment (turtlemoji) [inline src/containers/Modals/Review.tsx:48]: I recommend as a practice to use notations on comments that will need modifying the code later, like `TODO` or `NOTE` so it is easy to find later

**Review by turtlemoji — COMMENTED** [review]
lgtm besides some trivial comments!

File: package.json (hunk 1 above)
Comment (0xArdy) [reply package.json:32]: same benefits of downloading the font but faster configuration
https://fontsource.org/docs/getting-started/introduction

File: src/components/MadeByWonderland.tsx (hunk 2 above)
Comment (0xArdy) [reply src/components/MadeByWonderland.tsx:44]: wonderland svg logo has padding :T

File: src/containers/BridgeCard/TokenSection.tsx (hunk 3 above)
Comment (0xArdy) [reply src/containers/BridgeCard/TokenSection.tsx]: done https://github.com/defi-wonderland/superchain-bridge-ui/pull/34/commits/10a9ad819eb3581fc225eb3775bc9829182970a8

**Review by turtlemoji — APPROVED** [review]

**Pull request merged**